saleor/static/dashboard-next/components/AppLayout/AppLayout.tsx

```diff
@@ -12,12 +12,15 @@ import {
   withStyles,
   WithStyles
 } from "@material-ui/core/styles";
+import { CSSProperties } from "@material-ui/core/styles/withStyles";
+import Switch, { SwitchClassKey, SwitchProps } from "@material-ui/core/Switch";
 import * as classNames from "classnames";
 import * as React from "react";
 import SVG from "react-inlinesvg";
 import { RouteComponentProps, withRouter } from "react-router";
 
-import * as saleorLogo from "../../../images/logo-document.svg";
+import * as saleorLightLogo from "../../../images/logo-document.svg";
```

Comment:
Nitpick: could we use more meaningful names for logo files e.g. something like "logo-light-theme" / "logo-dark-theme"? About the logo in the dark mode - I've noticed that it's all white while on mockups the bar over "S" should be blue.

Reply:
Oopsie!
Of course we can.